MAINTAINERS.yml

```diff
@@ -0,0 +1,1166 @@
+        - drivers/ieee802154/
+
+"Drivers: Interrupt controllers":
+    status: orphaned
+    files:
+        - drivers/interrupt_controller/
+        - dts/bindings/interrupt-controller/
+        - include/drivers/interrupt_controller/
+        - include/dt-bindings/interrupt-controller/
+
+"Drivers: IPM":
+    status: maintained
+    maintainers:
+        - andrewboie
+    files:
+        - drivers/ipm/
+    description: >-
+        Inter-processor mailboxes
+
+"Drivers: kscan":
+    status: orphaned
+    files:
+        - drivers/kscan/
+        - include/drivers/kscan.h
+        - samples/drivers/espi/
+        - tests/drivers/kscan/
+
+"Drivers: LED":
+    # Provisional entry, subject to approval
+    status: maintained
+    maintainers:
+        - Mani-Sadhasivam
+    files:
+        - drivers/led/
+        - include/drivers/led/
+        - include/drivers/led.h
+        - samples/drivers/led_*/
+    labels:
+        - "area: LED"
+
+"Drivers: LED Strip":
+    status: maintained
+    maintainers:
+        - mbolivar-nordic
+    files:
+        - drivers/led_strip/
+        - dts/bindings/led_strip/
+        - include/drivers/led_strip.h
+
+"Drivers: lora":
+    # Provisional entry, subject to approval
+    status: maintained
+    maintainers:
+        - Mani-Sadhasivam
+    files:
+        - drivers/lora/
+        - include/drivers/lora.h
+        - samples/drivers/lora/
+
+"Drivers: Modem":
+    status: maintained
+    maintainers:
+        - mike-scott
+    files:
+        - drivers/modem/
+    labels:
+        - "area: Modem"
+
+"Drivers: Neural networking":
+    status: orphaned
+    files:
+        - drivers/neural_net/
+
+"Drivers: PCI":
+    status: maintained
+    maintainers:
+        - dcpleung
+    collaborators:
+        - andrewboie
+    files:
+        - drivers/pcie/
+        - include/drivers/pcie/
+    labels:
+        - "area: PCI"
+
+"Drivers: peci":
+    status: orphaned
+    files:
+        - drivers/peci/
+        - include/drivers/peci.h
+        - samples/drivers/peci/
+
+"Drivers: Pinmux":
+    status: maintained
+    maintainers:
+        - mnkp
+    files:
+        - doc/reference/peripherals/pinmux.rst
+        - drivers/pinmux/
+        - include/drivers/pinmux.h
+    labels:
+        - "area: Pinmux"
+
+"Drivers: PTP Clock":
+    status: orphaned
+    collaborators:
+        - jukkar
+    files:
+        - drivers/ptp_clock/
+        - include/ptp_clock.h
+
+"Drivers: PWM":
+    # Provisional entry, subject to approval
+    status: maintained
+    maintainers: anangl
+    files:
+        - drivers/pwm/
+        - dts/bindings/pwm/
+        - include/drivers/pwm.h
+        - tests/drivers/pwm/
+    labels:
+        - "area: PWM"
+
+"Drivers: Serial/UART":
+    status: maintained
+    maintainers:
+        - dcpleung
+    files:
+        - drivers/serial/
+        - dts/bindings/serial/
+    labels:
+        - "area: UART"
+
+"Drivers: Sensors":
+    status: maintained
+    maintainers:
+        - MaureenHelm
```

Comment:
collaborators:
        - avisconti

Reply:
I discussed some time ago with @MaureenHelm 
I can be a collaborator here.

Reply:
sure, i'll update this already.

Reply:
done now, @avisconti

Reply:
@ioannisg 
Thanks